package.json

```diff
@@ -172,6 +172,7 @@
     "filesize": "^3.6.1",
     "fs-extra": "^0.30.0",
     "get-stdin": "^5.0.1",
+    "git-clone": "^0.1.0",
```

Comment:
All it does, is spawning `git` comand.

I think it'll be better to rely on generic `child-process-ext/spawn` which we already use and which provides promise support and eventual debug logging out of a box

Reply:
Good idea.